src/main/java/core/basesyntax/services/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.services.ShopService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
```

Comment:
Avoid starting the class implementation with an empty line. Please remove the empty line before the class declaration.